lib/response-handlers/meal-response-handlers.js

```diff
@@ -0,0 +1,18 @@
+const $ = require('jquery')
+
+export function renderMeals(meals) {
+  for(var i = 0; i < meals.length; i++) {
+    addMealTable(meals[i]);
+    for(var j = 0; j < meals[i].foods.length; j++) {
+      addFoodRowsToMeal(meals[i], meals[i].foods[j]);
+    }
+  }
+}
+
+function addMealTable(meal) {
+  $('#meals-tables').append(`<table data-meal-id="${meal.id}" id="${meal.name.toLowerCase()}"><tr><th>${meal.name}</th><th>Calories</th></tr></table>`)
```

Comment:
Might be helpful to make append's argument a multi-line string for readability. But not something to request a change over